internal/ext/testdata/import_yaml_stream.yml

```diff
@@ -0,0 +1,84 @@
+flags:
```

Comment:
can you put one of these under a different namespace?

Reply:
@markphelps Sure, will do!